contrib/test/integration/e2e-base.yml

```diff
@@ -61,3 +61,19 @@
 
 - name: Enable iptables NAT for the bridge
   command: sysctl -w net.bridge.bridge-nf-call-iptables=1
+
+- name: Change /var label
+  command: 'semanage fcontext -a -R -t container_file_t /var/'
```

Comment:
If you want to just label everything under /var as container_file_t, for this boot, just do
chcon -t container_file_t -R /var

Don't do semanage or restorecon.

Reply:
fixed